projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -719,7 +720,7 @@ public function render_select_field( $id, $label, $value, $class, $required, $re
 		$field .= "\t<select name='" . esc_attr( $id ) . "' id='" . esc_attr( $id ) . "' " . $class . ( $required ? "required aria-required='true'" : '' ) . ">\n";
```

Comment:
You should probably add `tabindex='-1'` in here. Otherwise tabbing through the form is confusing as it focuses this hidden field before the visible dropdown.

Also, do we need to do anything to prevent screen readers from reading it?

Reply:
Well spotted! ✅